examples/with_winit/src/test_scene.rs

```diff
@@ -75,13 +73,24 @@ pub fn render_svg(sb: &mut SceneBuilder, svg: &PicoSvg) {
     }
 }
 
-#[allow(unused)]
-pub fn render_tiger(sb: &mut SceneBuilder) {
-    use super::pico_svg::*;
-    let xml_str =
-        std::str::from_utf8(include_bytes!("../../assets/Ghostscript_Tiger.svg")).unwrap();
-    let svg = PicoSvg::load(xml_str, 6.0).unwrap();
-    render_svg(sb, &svg);
+pub fn render_svg_scene(
+    sb: &mut SceneBuilder,
+    scene: &mut Option<SceneFragment>,
+    xform: Affine,
+    svg: &str,
+) {
+    if scene.is_none() {
```

Comment:
Consider `Option::get_or_insert_with()` here, it avoids the unwrap below.

Reply:
Nice, I wasn't aware of that API.